bin/algosim.py

```diff
@@ -164,6 +163,18 @@ def process_line(order_book, line, output=True):
                     if line['type'] == 'cancel':
                         order_book.cancel_order(line['side'],
                                                 line['order_id'])
+                    elif line['type'] == 'cancel_all':
+                        if line['side'] == "bid":
```

Comment:
you would need to decide if you use double or single quotes convention @joequant @dyn4mik3  :)